mlflow/protos/model_registry.proto

```diff
@@ -13,6 +13,7 @@ option (scalapb.options) = {
 };
 
 service ModelRegistryService {
+  // Note:: Experimental: This API may change or be removed in a future release without warning.
```

Comment:
is it not enough to just have this without putting experimental on every endpoint? (does it look weird in the generated doc?)

Reply:
Does not annotate on the service since a service is entity-non-grata in docs. Adding it for each API will guarantee that each API is annotated.